web/src/app/app.component.html

```diff
@@ -1,7 +1,7 @@
 <mat-sidenav-container class="sidenav-container">
   <mat-sidenav #sidenav [mode]="sideNavMode" class="sidenav" [ngClass]="{'sidenav__open': menuTriger}" fixedInViewport="false" role="navigation" [opened]="!isSmallScreen">
     <div class="heading" (click)="openMenu($event)">
-      <mat-icon svgIcon="logo_icon" class="heading__logo"></mat-icon>
+      <img src="assets/icons/Logo.svg" class="heading__logo"/>
```

Comment:
`Logo.svg` should be lowercase `logo.svg`

Reply:
done